### ISSUE TITLE
Trying to add shadcn-sveelte to a Project🐛 

### ISSUE THREAD

**Issue by rhscjohn-dev** [SvelteLab/SvelteLab#297]
### Describe the bug

While I was trying to add [shadcn-svelte](https://www.shadcn-svelte.com/docs/installation/sveltekit) to a new project,  the following command failed.
`npx shadcn-svelte@latest init`

### Reproduction

Step 1.  Set default template to Typescript Tailwind
Step 2.  Start New Project
Step 3.  Stop dev server.
Step 4.  Run the CLI(shadcn-svelte): npx shadcn-svelte@latest init
Step 4.1 You will be asked a few questions to configure components.json.
Step  4.1 aborts
Step 5 Start adding components to your project

### Expected behavior

There are about 6 questions that the CLI will prompt to properly initialize shadcn-svelte in your svelteKit project.

### Screenshots

![Screenshot 2024-02-21 152342](https://github.com/SvelteLab/SvelteLab/assets/60523782/2bcadcff-3d43-4d57-91e8-3df6903aaa59)

### Additional context

Not sure is this is a SvelteLab problem or a shadcn-problem.

**Comment by paoloricciuti** [comment]
If I have to guess I think it might be a problem with the web containers...basically everything that needs to enter the web container needs a specific set of headers. It might not be but I'm at least 34% sure it's that (lol)

**Comment by filiabel** [comment]
Working for me 🤷‍♂️
![image](https://github.com/user-attachments/assets/b091511b-689c-49a5-a454-3ed3a72415fe)

**Comment by SarcevicAntonio** [comment]
Yeah works for me too! I guess they changed something on their side to make it work. Feel free to repone if there is still an issue.
